liwords-ui/src/admin/tourney_editor.tsx

```diff
@@ -121,17 +130,20 @@ const SettingsModalForm = (mprops: {
 };
 
 export const TourneyEditor = (props: Props) => {
-  const [description, setDescription] = useState("");
-  const [disclaimer, setDisclaimer] = useState("");
-  const [name, setName] = useState("");
-  const [color, setColor] = useState("");
-  const [logo, setLogo] = useState("");
   const [settingsModalVisible, setSettingsModalVisible] = useState(false);
   const [selectedGameRequest, setSelectedGameRequest] = useState<
     GameRequest | undefined
   >(undefined);
   const [form] = Form.useForm();
+
+  const description = useWatch("description", form);
```

Comment:
Since we're not using controlled state anymore, this hook lets us watch for changes to the fields we care about for the sidebar preview. Otherwise it doesn't update as users type